airflow_dag_samples/embedded_dataset_dag/example_dag.py

```diff
@@ -10,7 +10,7 @@
 SOURCE = "{{ ti.xcom_pull(task_ids='create_volume') }}"
 dataset_filepath = 'dataset_example.jsonl'
 filename = 'sample_dataset'
-models_to_train = '["named_entity_recognition"]'
+models_to_train = '["NAMED_ENTITY_RECOGNITION"]'
```

Comment:
@NMPowell99 I think this change should be to `NlpModels.NAMED_ENTITY_RECOGNITION` to use the constant rather than using a hardcoded string here.

Reply:
Trying to import NlpModel from the sdk gives a dag import error when this dag is uploaded, so I instead added it to the sdk_calls file that this dag is referencing, with `models_to_train = [getattr(NlpModel, model) for model in models_to_train]`. I realize this is not ideal, so if you have an idea of how to get NlpModel into the actual dag file without an import error let me know!

Reply:
That makes sense to me, then should this be the lowercase still or is the uppercase change correct here?

Reply:
The upper case is correct/necessary for that like in the sdk_calls, unless I change it to use `.upper` or something